backend/de.metas.business/src/main/java/de/metas/invoice/service/IInvoiceBL.java

```diff
@@ -423,4 +424,6 @@ void registerLineCopyHandler(
 	 * - set Price_UOM_ID to C_InvoiceLine.C_UOM_ID
 	 */
 	void ensureUOMsAreNotNull(@NonNull InvoiceId invoiceId);
+
+	void setInvoiceLineTaxes(Properties ctx, de.metas.adempiere.model.@NonNull I_C_Invoice invoice);
```

Comment:
why `ctx` is needed. For legacy APIs u can extract it from `invoice`.

Reply:
✔️